### ISSUE TITLE
Google WebRisk API analyzer

### ISSUE THREAD

**Issue by mlodic** [intelowlproject/IntelOwl#54]
https://cloud.google.com/web-risk

We could create an analyzer for the WebRisk API. In this way, we can check a specific URL against the provided blacklists. This is the commercial version of the "GoogleSafeBrowsing" APIs.

**Comment by Sjsingh101** [comment]
Hi @mlodic, I would like to work on this. Will soon follow-up with the PR.

**Comment by mlodic** [comment]
Hey @Sjsingh101, thank you for your interest! Did you manage to get access to the WebRisk APIs?

**Comment by Sjsingh101** [comment]
https://cloud.google.com/web-risk/pricing

We have 2 APIs under this WebRisk APIs i.e. Update and Lookup. Lookup is free for the first 100,000 calls/month and Update is a paid service.
What I can do is to implement the Lookup and Update and then I can test the Lookup with a free quota. What do you suggest?

**Comment by mlodic** [comment]
Ok! the point is: did you manage to have an API key from Google?

From https://cloud.google.com/web-risk/docs/quickstart:
```
To gain access to Web Risk, you need to be added to the Web Risk allowlist. To be added to the allowlist, fill out the Web Risk sign-up form.
```

**Comment by Sjsingh101** [comment]
I have applied for that, but my approval is still pending.

Till then, could you please point to some other beginner-friendly issue to start with?

**Comment by mlodic** [comment]
You can look at the ones tagged with `beginner-friendly` and choose one

**Comment by mlodic** [comment]
solved with #417
